src/index.html

```diff
@@ -15,8 +15,23 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html"><img  src="images/logo.png" alt="Moyo" > </img></a>
```

Comment:
The `img` tag is self-closing, so you don't need to add `</img>` at the end. You can just write `<img src="images/logo.png" alt="Moyo" />`.